src/argilla/client/feedback.py

```diff
@@ -498,20 +488,18 @@ def add_records(
 
         for record in records:
             try:
-                record.fields = self.__fields_schema.parse_obj(record.fields)
+                self.__fields_schema.parse_obj(record.fields)
```

Comment:
This line, is just to apply the fields_schema validation, right?

Reply:
Yes, since before we were mapping the `FeedbackRecord.fields` to `FieldsSchema`, but we were calling `construct` over `FeedbackRecord` instead of creating it as `FeedbackRecord(...)`, so it was still a Python dict, adding that doesn't make too much sense for the moment, so we're just validating the `fields` based on `FeedbackDataset.fields` auto-generated `pydantic` schema